files/setup-06-vsphere-sources.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# Copyright 2023 VMware, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-2
+
+# Setup vSphere Sources
+
+set -euo pipefail
+
+echo -e "\e[92mCreating vSphere Secret ..." > /dev/console
+# Create vSphere Secret
+if [ ${VCENTER_DISABLE_TLS} == "True" ]; then
+    kn vsphere auth create \
```

Comment:
Bit weird that you deploy most of it using YAML/ytt and here use a CLI. Why not use manifests?

Reply:
I started out w/vSphere Sources which provided kn plugin ... I guess looking at Horizon Sources, I had no choice but perhaps might be good to just keep it consistent. I wasn't sure if there was a preference to leverage the plugin over manifests

Reply:
Yeah, no benefit I guess in using the CLI, so let's go for consistency.

Reply:
Resolved